features/testbot/trip.feature

```diff
@@ -48,12 +65,85 @@ Feature: Basic trip planning
             | kl    |
             | la    |
 
+        When I plan a trip I should get
+            | waypoints               | trips         |
+            | a,b,c,d,e,f,g,h,i,j,k,l | alkjihgfedcba |
+
+    Scenario: Testbot - Trip: Roundtrip FS waypoints (more than 10)
+        Given the node map
+            """
+            a b c d
+            l     e
+            k     f
+            j i h g
+            """
+
+        And the ways
+            | nodes |
+            | ab    |
+            | bc    |
+            | de    |
+            | ef    |
+            | fg    |
+            | gh    |
+            | hi    |
+            | ij    |
+            | jk    |
+            | kl    |
+            | la    |
+
+        When I plan a trip I should get
+            | waypoints               | trips         | source |
```

Comment:
@TheMarex I asked @ghoshkaj to move `source` and `dest` into the Results table - they're query parameters in the same way that `waypoints` are.

Having these in the `When` table allows multiple combinations for the same scenario - putting them in the `Given the query options` clause limits us to one set per scenario.